app/models/spree_amazon/configuration.rb

```diff
@@ -0,0 +1,10 @@
+class SpreeAmazon::Configuration < Spree::Preferences::Configuration
+  preference :client_id, :string
+  preference :merchant_id, :string
+  preference :aws_access_key_id, :string
+  preference :aws_secret_access_key, :string
+
+  def use_static_preferences!
+    raise "SpreeAmazon::Configuration cannot use static preferences"
+  end
```

Comment:
That's why in the `spree.rb` [template](https://github.com/solidusio/solidus/blob/60ee1dffc7c3932cd7652cc2b4566f08a12f74b8/core/lib/generators/spree/install/templates/config/initializers/spree.rb#L4):

```ruby
  # Without this preferences are loaded and persisted to the database. This
  # changes them to be stored in memory.
  # This will be the default in a future version.
  # This line resets all preferences! It should be the first line in the block
  config.use_static_preferences!
```

Great, thanks again!